### PR TITLE
Revert "Ltm resync from 1.6 develop with drift manager changes"

### PR DESCRIPTION
**Pull request by smk4664**
Reverts nautobot/nautobot-app-welcome-wizard#89

The above PR had added Nautobot 2.0 support to the ltm-1.6 branch, but the intention was to just update to meet drift manager. Since Drift Manager does not currently support ltm-1.6 and these changes break ltm-1.6 I am reverting the full PR instead of trying to fix this to support Nautobot 1.6 again.

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -231,7 +227,7 @@ jobs:
       - "publish_pypi"
     runs-on: "ubuntu-20.04"
     env:
-      SLACK_WEBHOOK_URL: "${{ secrets.SLACK_WEBHOOK_URL }}"
+      SLACK_WEBHOOK_URL: "${{ '{{ secrets.SLACK_WEBHOOK_URL }}' }}"
```
Comment (gsnider2195) [inline .github/workflows/ci.yml:230]: Was this intentional?

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -241,12 +237,11 @@ jobs:
       - name: "Send a notification to Slack"
         # ENVs cannot be used directly in job.if. This is a workaround to check
         # if SLACK_WEBHOOK_URL is present.
-        if: "env.SLACK_WEBHOOK_URL != ''"
+        if: "${{ env.SLACK_WEBHOOK_URL != '' }}"
```
Comment (gsnider2195) [inline .github/workflows/ci.yml:240]: I think this was correct before

File: .github/workflows/ci.yml [hunk 3]
```diff
@@ -241,12 +237,11 @@ jobs:
...
         uses: "slackapi/slack-github-action@v1.17.0"
         with:
           payload: |
             {
-              "text": "${{ env.SLACK_MESSAGE }}",
```
Comment (gsnider2195) [inline .github/workflows/ci.yml:305]: Why remove this?

File: tasks.py [hunk 4]
```diff
@@ -38,7 +44,7 @@ def is_truthy(arg):
 namespace.configure(
     {
         "welcome_wizard": {
-            "nautobot_ver": "latest",
+            "nautobot_ver": "1.5.5",
```
Comment (gsnider2195) [inline tasks.py:47]: Probably should be at least 1.6.0 here?

File: .github/workflows/ci.yml (hunk 1 above)
Comment (smk4664) [reply .github/workflows/ci.yml:230]: Yes, this matches the LTM-1.6 from Golden Config.

File: tasks.py (hunk 4 above)
Comment (smk4664) [reply tasks.py:47]: 1.6 drops support for python 3.7. In order to keep the existing Pyproject.toml, I changed this to 1.5.5. I tried with 1.6.0, but I get dependency resolution issues in the docker build stage.

File: .github/workflows/ci.yml (hunk 3 above)
Comment (smk4664) [reply .github/workflows/ci.yml:305]: It's a little further below in the text under the blocks. This is copied from nautobot-app-golden-config:ltm-1.6

File: .github/workflows/ci.yml (hunk 1 above)
Comment (gsnider2195) [reply .github/workflows/ci.yml]: Is this a syntax you're familiar with? I'm not sure I understand how this works.

File: tasks.py (hunk 4 above)
Comment (gsnider2195) [reply tasks.py:47]: Are we still supposed to be supporting python 3.7 for this project? It was EOL a year ago.

File: .github/workflows/ci.yml (hunk 1 above)
Comment (smk4664) [reply .github/workflows/ci.yml]: I reverted it, I think the source I got this from might be incorrect.

File: tasks.py (hunk 4 above)
Comment (smk4664) [reply tasks.py:47]: I don't believe we will be creating another release of Welcome Wizard for 1.6. I was trying to make the minimum number of changes required to get Upstream Testing to work.

Comment (smk4664) [reply tasks.py:47]: I retested, I got this to work with 1.6, which pulls the latest 1.6.x branch. It is possible that 1.6.0 didn't work, or something else changed as well.

**Review by gsnider2195 — APPROVED** [review]

**Pull request merged**